.circleci/config.yml

```diff
@@ -16,32 +16,44 @@ workflows:
               ignore: /.*/
 
 jobs:
-  test: # runs not using Workflows must have a `build` job as entry point
+  test:
+    resource_class: xlarge
```

Comment:
The build process for the tests is killed if using anything smaller than this ...